storage/aptosdb/src/pruner/transaction_store/test.rs

```diff
@@ -102,14 +105,25 @@ fn verify_txn_store_pruner(
         &txns,
     );
 
+    let batch = SchemaBatch::new();
+    for i in 0..=num_transaction as u64 {
+        let usage = StateStorageUsage::zero();
+        batch
+            .put::<VersionDataSchema>(&i, &usage.into())
+            .expect("Must succeed.");
```

Comment:
🤣  come on, it's a test, just `unwrap()`

Reply:
I felt it was weird when writing `expect`, but my brain was stuck at that time and couldn't find the `unwrap` to use. 🤣